Tests/Core/CheckoutAPIClientTests.swift

```diff
@@ -120,4 +122,33 @@ class CheckoutAPIClientTests: XCTestCase {
         wait(for: [expectation], timeout: 1.0)
     }
 
+    func testLoggerInitialisation() {
+
+        let stubSDKAppIDProvider = StubSDKAppIDProvider()
+        stubSDKAppIDProvider.sdkAppIDToReturn = .success(.init(version: "version_string", appId: "stub_app_id"))
+
+        let stubLogger = StubLogger()
+
+        _ = CheckoutAPIClient(publicKey: "pk_test_6ff46046-30af-41d9-bf58-929022d2cd14",
+                              environment: .sandbox,
+                              jsonEncoder: JSONEncoder(),
+                              jsonDecoder: JSONDecoder(),
+                              requestorAppBundle: Foundation.Bundle.main,
+                              uiDevice: UIKit.UIDevice.current,
+                              sdkAppIDProvider: stubSDKAppIDProvider,
+                              logger: stubLogger)
+
+        XCTAssertNil(stubLogger.enableLocalProcessorCalledWith)
+        XCTAssertEqual(stubLogger.enableRemoteProcessorCalledWith?.environment, .sandbox)
+        // this check is dependent on RemoteProcessorMetadata implementing Equatable
+//        XCTAssertEqual(stubLogger.enableRemoteProcessorCalledWith?.remoteProcessorMetadata, RemoteProcessorMetadata(productIdentifier: <#String#>,
```

Comment:
Can we remove this commented-out code?

Reply:
This will be uncommented once I update the version of `CheckoutEventLoggerKit` used with `Frames`